.github/workflows/ci-cd.yml

```
@@ -1,318 +1,129 @@
+          SERVICE_IP=$(kubectl get service simple-app-service -n ${{ env.KUBE_NAMESPACE }} -o jsonpath='{.spec.clusterIP}' || echo "N/A")
+          echo "Service IP: $SERVICE_IP"
+          echo "Production deployment completed successfully!"
+          echo ""
+          echo "🚀 To access production locally:"
+          echo "  Method 1: Use the helper script"
+          echo "    ./scripts/port-forward.sh production"
+          echo ""
+          echo "  Method 2: Manual port forwarding"
+          echo "    kubectl port-forward service/simple-app-service 3002:80 -n production"
+          echo ""
+          echo "  Then visit: http://localhost:3002" 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)